blazor/rich-text-editor/toolbar.md

```diff
@@ -21,7 +21,7 @@ You can customize the toolbar configurations by usig the [RichTextEditorToolbarS
 
```

Comment:
Remove the toolbar image for this topic

https://github.com/syncfusion-content/blazor-docs/blob/f39a29f71966edb56938fcf18cab19a948d3cf88/blazor/rich-text-editor/images/blazor-richtexteditor-toolbar-types.png